src/main/java/org/sonar/plugins/stash/PluginUtils.java

```diff
@@ -6,13 +6,21 @@
 import java.util.Enumeration;
 import java.util.jar.Attributes;
 import java.util.jar.Manifest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class PluginUtils {
 
     // Hiding implicit public constructor with an explicit private one (squid:S1118)
     private PluginUtils() {
     }
-    
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PluginUtils.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)